### PR TITLE
Unique solid name

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Related to https://github.com/omichel/webots/pull/6392.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
Given that I have to update some of the tutorials screenshots because of the changed solid name, I will also update the Webots version and toolbar appearance in the tutorials screenshots.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by omichel** [comment]
:eyes:

File: reference/solid.md [hunk 1]
```diff
@@ -149,3 +151,56 @@ that the center of mass of the [Solid](#solid) does not depend on its
...
 of the [Physics](physics.md) node (in coordinates relative to the center of the
 [Solid](#solid)).
+
+### Unique Solid name
+
+The capability of identifying uniquely each [Solid](#solid) node is a base requirement for other advanced features,
+for example to let the viewpoint follow a solid object or to store the preferences for the rendering devices overlays.
+For this reason since Webots R2018a, the user is encouraged to set unique names for sibling [Solid](#solid) nodes and
+a warning is printed in the console if this rule is not respected.
+With sibling [Solid](#solid) nodes we indicate two [Solid](#solid) nodes for which the upper [Solid](#solid) node is the same.
```
Comment (omichel) [inline reference/solid.md]: I would use "mean" instead of "indicate two".

**Comment by stefaniapedrazzi** [comment]
I changed the verb as you suggested.

**Pull request merged**